mosdef_cassandra/writers/inp_functions.py

```diff
@@ -265,7 +265,7 @@ def generate_input(
     boxes = []
     for box in system.boxes:
         if isinstance(box, mbuild.Compound):
-            box_dims = np.hstack((box.periodicity, box.boundingbox.angles))
+            box_dims = np.hstack((box.box.lengths, box.box.angles))
```

Comment:
Doesn't the new `mbuild.Box` class support conversions to a box matrix? Can we use that instead? Then we can strip out our box matrix helper functions. I think the `mbuild.Box` class has the box vectors in rows, so we need to watch out for that.

Reply:
Yes.  I just made that change.  Should we remove your TODO comment now?

Reply:
Well, I corrected for the row vectors, but apparently `convert_to_boxmatrix` gives row vectors too, and it is only manually transposed in `get_box_info`, so I will go back and un-transpose the matrices from mbuild.

Reply:
Sorry!

Can we remove the `convert_to_boxmatrix` function from MoSDeF Cassandra all together? I can't imagine we use it anywhere else?

Reply:
Sure.  I had actually already done so but reverted it because I wasn't sure you'd want it completely gone.